cluster/health-monitor.sh

```diff
@@ -1,51 +0,0 @@
-#!/bin/bash
```

Comment:
Seems odd to remove this... was/is this only useful for GCE?

Reply:
Currently in the code base, only GCE is using this.

However, I think what you said makes sense, there might be someone using this script. Given so, I think I'll leave the file here for one more release, and file an issue to remove it next release.

Reply:
Done.